### ISSUE TITLE
<ctrl><f> triggers find dialog but the title of the dialog is [JavaScript Application]

### ISSUE THREAD

**Issue by MikeRatcliffe** [MikeRatcliffe/Acebug#5]
&lt;ctrl&gt;&lt;f&gt; triggers find dialog but the title of the dialog is [JavaScript Application]

**Comment by nightwing** [comment]
thats is default behaviour from ace
we should implement better find, like in cloud9

**Comment by MikeRatcliffe** [comment]
Exactly, this is just a todo really. I don't think there is a way to trigger the replace dialog either.

**Comment by MikeRatcliffe** [comment]
We should go through the following and make sure that each prompt is handled properly:
https://github.com/ajaxorg/ace/blob/master/lib/ace/commands/default_commands.js

Normal prompt windows cannot have titles but we can use the nsiPromptService from XUL:
https://developer.mozilla.org/en/XPCOM_Interface_Reference/nsIPromptService
